modules/lib.lua

```diff
@@ -260,14 +261,11 @@ if isServer then
             SetPedIntoVehicle(ped, veh, -1)
         end
 
-        local owner = lib.waitFor(function()
-            local owner = NetworkGetEntityOwner(veh)
-            if owner ~= -1 then return owner end
-        end, 5000)
-
+        Entity(veh).state:set('setVehicleProperties', props, true)
+        Entity(veh).state:set('initVehicle', true, true)
```

Comment:
```suggestion
        Entity(veh).state:set('initVehicle', true, true)
        Entity(veh).state:set('setVehicleProperties', props, true)
```

We need to intiate the vehicle before trying to set the properties